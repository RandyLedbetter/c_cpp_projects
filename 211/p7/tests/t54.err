Error: could not remove New_York
Error: could not remove Los_Angeles
