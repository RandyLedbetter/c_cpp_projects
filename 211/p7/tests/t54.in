insert Chico Sacramento 90
insert Chico Redding 73
insert Sacramento San_Francisco 87
insert Sacramento Los_Angeles 384
remove New_York
remove Los_Angeles
remove Los_Angeles
lookup Chico
lookup Sacramento
lookup Redding
