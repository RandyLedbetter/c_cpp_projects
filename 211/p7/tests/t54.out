Chico: none, Sacramento(90), Redding(73)
Sacramento: Chico(90), San_Francisco(87), none
Redding: Chico(73), none, none
